### ISSUE TITLE
Error unknown service

### ISSUE THREAD

**Issue by uhu313** [InventoCasa/ha-advanced-blueprints#59]
The automation "Surplus load" ("automation.overload") has an action that calls an unknown service: "pyscript.pv_excess_control"

this is the error. Can you help me? 

Everything is installed regarding instruction…

greeting uhu313
![IMG_1492](https://github.com/InventoCasa/ha-advanced-blueprints/assets/75754000/198ff8f1-da01-4402-b894-2c682e20d031)

**Comment by Ardarel** [comment]
Did you found solution?

**Comment by uhu313** [comment]
Yes. In fact, two lines of code were indented incorrectly and the script didn't like that… 😂

**Comment by nukleuz80** [comment]
Please show incorrect code...

**Comment by 123vvbznfjk321** [comment]
Hallo, kann mir jemand erklären wie man PV Excess Control nach der Installation bedient bzw. prüfen kann, ob die Automation läuft ?

**Comment by nukleuz80** [comment]
> Hallo, kann mir jemand erklären wie man PV Excess Control nach der Installation bedient bzw. prüfen kann, ob die Automation läuft ?

Das versuche ich auch gerade herauszufinden, doch ohne laufenden Blueprint wird das nichts...

**Comment by Maik7** [comment]
in configuration.yaml debug Infos einschalten
```

logger:
  default: warn
  logs:
    custom_components.pyscript.file.pv_excess_control: debug
```

und dann wird bei laufendem PC Exess Control minütlich in home-assistant.log der Status ausgegeben.

**Comment by 123vvbznfjk321** [comment]
Das funktioniert.Wie aber wird die Automation bei der Spülmaschine mit einem Shelly praktisch umgesetzt ? Shelly EIN schalten und Spülmaschine starten anschließend Shelly sofort wieder AUS schalten. Wenn es dann passt schaltet PV EXCESS wieder ein ?Am 08.05.2024 19:08 schrieb Maik7 ***@***.***>:
in configuration.yaml debug Infos einschalten

logger:
  default: warn
  logs:
    custom_components.pyscript.file.pv_excess_control: debug

und dann wird bei laufendem PC Exess Control minütlich in home-assistant.log der Status ausgegeben.

**Comment by nukleuz80** [comment]
Zum ursprünglichen Thema: ich musste meine pv_excess_control.py Datei neu hochladen. Läuft…
